.github/workflows/_build.yml

```diff
@@ -103,8 +142,9 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
   PercyTests:
+    if: (needs.CheckVisualTests.outputs.requires-visual == 'true')
```

Comment:
This will run all percy snapshots if any components change correct?

Reply:
Yes, as there are going to be some components that use other components, so we might need to test those too without crazy custom logic to check which are connected or not.

This is how we have done this in other projects in Skyscanner too, so its more just don't run if we change non component code!